recipes/graph-tool/build.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -x
+
+export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
+export CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include -std=c++17"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+
+export BOOST_ROOT="${PREFIX}"
+
+# Explicitly set this, which is used in configure.
+# (We patched away the auto-detection of this variable.)
+export BOOST_PYTHON_LIB=boost_python37
+
+# Note about PYTHON_LIBS:
+# If left unset, the configure script will set PYTHON_LIBS as follows:
+#
+#   PYTHON_LIBS="-L${PREFIX}/lib -lpython3.7m"
+#
+# ...but we never want to link against libpython in a conda environment.
+# That's because conda's python executable is STATICALLY linked.
+# It does not link against libpython3.7.dylib and therefore no
+# python extension modules should link against libpython3.7, either!
+# In fact, if we link against libpython, we'll end up with segfaults.
+# Instead, all python symbols will be loaded by python executable itself.
+#
+# So, we don't want to set PYTHON_LIBS, but we can't leave it empty.
+# This is a suitable no-op.
+PYTHON_LIBS=" "
+
+if [[ $(uname) == Darwin ]]; then
```

Comment:
Use `$target_platform == osx*` instead of `uname`

Reply:
done